VMBackup/main/Utils/HandlerUtil.py

```diff
@@ -532,10 +532,21 @@ def do_status_report(self, operation, status, status_code, message, taskId = Non
         sub_stat = []
         stat_rept = []
         self.add_telemetry_data()
+        snapshotCreatorInfo = ""
+
+        if "snapshotCreator" in HandlerUtility.telemetry_data.keys():
+            snapshotCreatorInfo = "{0}{1}={2}, ".format(snapshotCreatorInfo , "snapshotCreator" , HandlerUtility.telemetry_data["snapshotCreator"])
```

Comment:
Define strings "snapshotCreator", "hostStatusCodePreSnapshot" and "hostStatusCodeDoSnapshot" in commom constants. Use those constants here.